### PR TITLE
Add info about multiple --browser arguments

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Porting over changes from https://github.com/blackbaud/skyux2-docs/pull/821 so that PR checks will pass. ... Addresses https://github.com/blackbaud/skyux2-docs/issues/820.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments](https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments)

File: src/stache/includes/cli/browser.html [hunk 1]
```diff
@@ -1 +1 @@
...
...fies the browser to use. This option is useful for cross-browser manual testing. The available values are platform-dependent, and they correspond to the names of the browser executables on each platform. For example, the value for Chrome is <sky-code>google chrome</sky-code> on macOS, <sky-code>google-chrome</sky-code> on Linux, and <sky-code>chrome</sky-code> on Windows. Microsoft Edge is an exception that is not platform-dependent, and its value is <sky-code>edge</sky-code> on all platforms. You can also pass in the browser's full path. SKY UX passes the value to the <sky-code>open</sky-code> library as the <sky-code>app</sky-code> property, so for more information, see <a href="https://github.com/sindresorhus/open#app">the <sky-code>open</sky-code> library's README file</a>. You can specify multiple <sky-code>--browser</sky-code> arguments. The first argument specifies the browser to use, while subsequent arguments are passed to the browser. This is useful for serving the application in incognito/private mode. For example, to run the application in Chrome on Linux and use inincognito mode, enter <sky-code>skyux server --browser='google-chrome' --browser='--incognito'</sky-code>.
```
Comment (Blackbaud-BobbyEarl) [inline src/stache/includes/cli/browser.html]: Last sentence says `skyux server` instead of `skyux serve`.

Comment (blackbaud-johnly) [reply src/stache/includes/cli/browser.html]: Fixed the typo. Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments](https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments](https://developer.blackbaud.com/skyux?_sp.spa=pr-multiple-browser-arguments)

**Pull request merged**